tests/snippets/stdlib_os.py

```diff
@@ -161,3 +161,8 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 	assert os.path.basename(fname) == FILE_NAME
 	assert os.path.dirname(fname) == tmpdir
+
+# supports
+assert isinstance(os.supports_fd, set)
+assert isinstance(os.supports_dir_fd, set)
+assert isinstance(os.supports_follow_symlinks, set)
```

Comment:
Can you test that `os.stat` (or any other compatible function) is in `os.supports_*`?

Reply:
Nothing is insdie of the set for now. It is just placeholders. See fd_funcs which only includes `None` and `Some(false)`

Reply:
Cool. Did not see that.